.golangci.yml

```diff
@@ -254,8 +254,8 @@ issues:
         - forbidigo
         - godot
 
-    # Allow fmt.Printf() in lncli.
-    - path: cmd/lncli/*
+    # Allow fmt.Printf() in commands.
+    - path: cmd/commands/*
```

Comment:
This refactoring commit seems unnecessary in this PR. Would be simpler to do this sort of change once the saga is merged into `master` perhaps.

The commit body should explain why this change is necessary.

Reply:
Added this to the commit body:
```
We want to export some of our CLI code to re-use in other projects. But
in Golang you cannot import code from a `main` package.
So we need to move the actual code into its own package and only have 
the `func main()` in the `main` package.
```